storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -5892,9 +5891,9 @@ static void check_datadir_enctry_access(const char *name,
                                         const struct stat *statinfo) {
   const char *entry_type = S_ISDIR(statinfo->st_mode) ? "directory" : "file";
   if ((statinfo->st_mode & S_IRUSR) != S_IRUSR) {
-    msg("xtrabackup: Error: %s '%s' is not readable by "
-        "XtraBackup\n",
-        entry_type, name);
+    xb::error() << entry_type << " " << name
+                << " is not readable by "
+                   "XtraBackup";
```

Comment:
```suggestion
    xb::error() << entry_type << " '" << name
                << "' is not readable by "
                   "XtraBackup";
```

Reply:
@altmannmarcelo what about introducing quote() function? so we can use freely  << quote(name) <<

Reply:
@satya-bodapati looks like a more elegant way to solve the problem! Go for it.